proto/import_sstpb.proto

```diff
@@ -300,9 +300,15 @@ message KVMeta {
     // The file name of the KV file.
     string name = 1;
 
+    // file offset, sometimes only need to get a part of data from the merged file
+    uint64 offset = 11;
+
     // file length for check.
     uint64 length = 2;
 
+    // file length after compressed
+    uint64 compress_length = 12;
```

Comment:
Note: Can we extract those fields to another message? (In fact what we want to restore isn't SST at all in the context.)